google-common/src/main/scala/org/apache/pekko/stream/connectors/google/auth/GoogleOAuth2.scala

```diff
@@ -37,7 +37,7 @@ private[auth] object GoogleOAuth2 {
 
   private val oAuthTokenUrl = "https://oauth2.googleapis.com/token"
 
-  def getAccessToken(clientEmail: String, privateKey: String, scopes: Seq[String])(
+  def getAccessToken(clientEmail: String, privateKey: String, scopes: Set[String])(
```

Comment:
I think a mina filter will be needed. or add an overloaded method which use seq.toSet then?

Reply:
I'll wait till 1.0.2 is out since that will fix the existing MiMa issue and then I will see.if this is still problematic